plugin.go

```diff
@@ -167,8 +264,7 @@ func (p Plugin) Exec() error {
 		options := []slack.MsgOption{}
 		if text != "" {
 			options = append(options, slack.MsgOptionText(text, false))
-		}
-		if len(blocks) > 0 {
+		} else if len(blocks) > 0 {
```

Comment:
what happens if there are no blocks ?
what should it fall through to ?

Reply:
fixed.